### PR TITLE
TR-8: Create dotenv file config

### PR DESCRIPTION
**Pull request by vesear**

### CODE REVIEW

File: tests/authentication/logInPage.spec.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
 import { logInPage } from "../../pageObjects/auth/logInPage.js";
 import { expect } from "chai";
+import "dotenv";
```
Comment (mashadolya) [inline tests/authentication/logInPage.spec.js:3]: ????

File: tests/authentication/logInPage.spec.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+
     expect(await logInPage.isPageOpened()).to.be.true;
 
-    await logInPage.logIn(ADMIN.USERNAME, ADMIN.PASSWORD);
+    await logInPage.logIn(CONFIG.USERNAME, CONFIG.PASSWORD);
     const currentUserName = await logInPage.getUserName();
 
-    expect(currentUserName).to.be.eql(ADMIN.NAME + " " + ADMIN.SURNAME);
+    expect(currentUserName).to.be.eql(
+      process.env.NAME + " " + process.env.SURNAME
```
Comment (mashadolya) [inline tests/authentication/logInPage.spec.js:16]: Pls use values from CONFIG

File: config.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-  ADMIN: {
-    NAME: "Artyom",
-    SURNAME: "Veselko",
-    USERNAME: "veselkoartyom@gmail.com",
-    PASSWORD: "Alexis-1937",
-  },
-};
\ No newline at end of file
+  USERNAME: process.env.USER_NAME,
+  PASSWORD: process.env.PASSWORD,
```
Comment (mashadolya) [inline config.js:5]:
```
export const CONFIG = {
  USER:{  USERNAME: process.env.USER_NAME,
  PASSWORD: process.env.PASSWORD,
  NAME: process.env.NAME,
  SURNAME: process.env.SURNAME,},

};
```

**Review by mashadolya — CHANGES_REQUESTED** [review]

File: tests/authentication/logInPage.spec.js (hunk 1 above)
Comment (vesear) [reply tests/authentication/logInPage.spec.js:3]: ;(((

**Pull request merged**